src/main/java/edu/cornell/kfs/sys/batch/PurgeTablesStep.java

```diff
@@ -2,19 +2,29 @@
 
 import java.util.Date;
 
+import org.kuali.kfs.core.api.datetime.DateTimeService;
 import org.kuali.kfs.sys.batch.AbstractStep;
 
 import edu.cornell.kfs.sys.batch.service.TablesPurgeService;
 
 public class PurgeTablesStep extends AbstractStep {
     
+    private DateTimeService dateTimeService;
```

Comment:
Is the dateTimeService needed here? Or could this just use the one from AbstractStep?

Reply:
Good catch. Changing.